plugins/nvme/fu-nvme-device.c

```diff
@@ -390,6 +390,7 @@ fu_nvme_device_init (FuNvmeDevice *self)
 {
 	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_REQUIRE_AC);
 	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_UPDATABLE);
+	fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_NO_GUID_MATCHING);
```

Comment:
Is that right?  I was thinking this might be a failure that logical ID is wrong

Reply:
#2367

Reply:
I think FWUPD_DEVICE_FLAG_NO_GUID_MATCHING should have been the default, but alas it's too risky to change now.

Reply:
>  I was thinking this might be a failure that logical ID is wrong

Leaving NO_GUID_MATCHING turned off means we don't compare the physical or logical IDs at all. It's just a bad default IMHO.